src/src/targets/EWRF_E7082VM/EWRF_E7082VM.c

```diff
@@ -63,6 +63,30 @@ uint8_t amp_state = 0;
                                       { 820,   830,  830,  840,  860,  890,  890,  895,  895}, // 100 mW
                                       {1450,  1450, 1450, 1450, 1450, 1450, 1450, 1450, 1450}  // ~400 mW
                                       };
+#elif defined(A03)
```

Comment:
Wrong define.

Reply:
Sorry, the first time I submitted PR, there were many omissions. this problem has been corrected.